app/assets/stylesheets/go.scss.erb

```diff
@@ -0,0 +1,91 @@
+.go{
+    margin: auto !important;
+    max-width: 900px !important;
+    padding: 1em !important;
+
+    .board{
+        background: url(<%= asset_path("grid.svg") %>);
+        padding-bottom: 100%;
+        width: 100%;
+        margin: 1em 0;
+    }
+
+    button{
+        font-size: 13px;
+    }
+
+    .board button{
```

Comment:
Since we're using sass, let's nest the selectors for clarity.  I think that means that this can go inside the `.board` section abover.